enzyme/CMakeLists.txt

```diff
@@ -70,17 +71,29 @@ message("CMAKE_PREFIX_PATH " ${CMAKE_PREFIX_PATH})
 
 find_package(LLVM REQUIRED CONFIG)
 
+if (NOT LLVM_ENABLE_PLUGINS)
+  message("LLVM_ENABLE_PLUGINS OFF")
+  message("Setting ENZYME_ENABLE_PLUGINS OFF")
+  set(ENZYME_ENABLE_PLUGINS OFF)
+endif()
+
+if ((NOT ENZYME_EXTERNAL_SHARED_LIB) AND (NOT ENZYME_ENABLE_PLUGINS))
+  message("ENZYME_EXTERNAL_SHARED_LIB OFF")
+  message("ENZYME_ENABLE_PLUGINS OFF")
+  message(SEND_ERROR "Building neither plugins nor a shared library. This is likely not what you want.")
```

Comment:
i think we need to make this more instructive

"You have selected a built without creating the Enzyme plugin, nor Enzyme as a shared library -- and nothing will be built."

Reply:
@wsmoses pushed.